.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Keycloak CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Compute cache restore key
+        # Always recompute on a push so that the maven repo doesnt grow indefinitely with old versions
+        run: |
+          if ${{ github.event_name == 'pull_request' }}; then echo "::set-env name=COMPUTED_RESTORE_KEY::q2maven-"; fi
+      - name: Cache Maven Repository
+        id: cache-maven
+        uses: n1hility/cache@v2
```

Comment:
The [GH actions documentation](https://help.github.com/en/actions/language-and-framework-guides/building-and-testing-java-with-maven#caching-dependencies) suggests something like:

```
- name: Cache Maven packages
    uses: actions/cache@v1
    with:
      path: ~/.m2
      key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
      restore-keys: ${{ runner.os }}-m2
```

Not sure what is the benefit of `n1hility/cache@v2` but I'd assume that https://github.com/marketplace/actions/cache is the GH supported version.

Reply:
I stole this from Quarkus - I'm not sure either. Looks like it's working fine though.

Reply:
From what I tried, yes. I was wondering if would be better to use the official release, but that's not a deal breaker for me.

Reply:
I just assumed that Quarkus folks had some good reason, but happy to just use the default as well. Let's switch to that